tests/cdi/cdi_suite_test.go

```diff
@@ -13,14 +13,20 @@ import (
 
 // template parameters
 const (
-	pvcEPHTTPNOAUTHURL = "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
-	invalidPVCURL      = "https://noneexist.com"
-	pvcName            = "golden-pvc"
-	pvcName1           = "golden-pvc1"
-	vmName             = "test-vm"
-	vmAPIVersion       = "kubevirt.io/v1alpha2"
-	rawPVCFilePath     = "tests/manifests/golden-pvc.yml"
-	rawVMFilePath      = "tests/manifests/test-vm.yml"
+	cirrosURL    = "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
+	invalidURL   = "https://noneexist.com"
+	emptyURL     = ""
+	vmAPIVersion = "kubevirt.io/v1alpha2"
+)
+
+const (
+	rawDataVolumePath = "tests/manifests/datavolume.yml"
+	rawPVCFilePath    = "tests/manifests/golden-pvc.yml"
+	rawVMFilePath     = "tests/manifests/test-vm.yml"
+)
+
+const (
```

Comment:
nit and it's a matter of personal style, but these 3 constants blocks could be combined.

Reply:
Actually I personally prefer use compact style, so I think you need to change back your code @gouyang wdyt ?

Reply:
@shiywang I don't understand the question for me? Could you explain a bit more?

Reply:
it means instead of split const
```
const (
a = 1
b = 2
) 

const (
c = 3
)
```
we prefer
```
const (
a = 1
b = 2
c = 3
)
```

Reply:
Group variables and constants are an idiom in go. I don't understand why you're going to anti it.